app/core/Tracker2/CookieConsentTracker.js

```diff
@@ -0,0 +1,74 @@
+import cookieconsent from 'cookieconsent'
+import 'cookieconsent/build/cookieconsent.min.css'
+
+import BaseTracker from './BaseTracker'
+
+/**
+ * Not a true tracker in that this tracker does not report data but instead
+ * prompts the user to consent to tracking.  It shares the same initialization
+ * and event emitter characteristics of a tracker.
+ */
+export default class CookieConsentTracker extends BaseTracker {
+  constructor (store) {
+    super()
+
+    this.store = store
+    this.loadStatus(undefined)
+  }
+
+  /**
+   * Requires that user locale has already been loaded on the page
+   */
+  _initializeTracker () {
+    if (!this.store.getters['me/inEU']) {
+      this.onInitializeSuccess()
```

Comment:
How do we ensure this runs after user locale has been loaded? Is there a check or assertion that can be made?

Reply:
We handle that as part of the tracker initialization in application.coffee here https://github.com/codecombat/codecombat/pull/5776/files#diff-328bf951d06f0baeb1c213ebb6b69f58R84

Added a check here and an error log, opted to not throw an error because the consent modal will still show in the event that we don't load the locale properly, just in the wrong language.  Could be convinced otherwise though ;)

Reply:
That sounds good! Happy to leave as is!